client/blocks/post-status/test/index.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import identity from 'lodash/identity';
+import Gridicon from 'components/gridicon';
+
+describe( 'PostStatus', ( ) => {
+	let PostStatus;
+
+	before( ( ) => {
+		PostStatus = require( '..' ).PostStatus;
```

Comment:
I think you can import this at top-of-file now. You'd only have needed it if you had some other test helpers which were prerequisites to be defined before the component itself (e.g. fake DOM, mockery, etc). Should help clean things up a bit.